### DISCUSSION TITLE
Github Desktop Client support page missing -- Data being potentially corrupted

### DISCUSSION THREAD

**Discussion by ar-visions** [community/community#54589] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Summation of 15-20 minutes to find this page:
There is no formal place to write bug reports for Github Desktop.  I assume this to mean Github is not interested in improving their product.  The managerial hand is clearly at play when you navigate through 30 modals just to get to this what seems like a Github fanclub page.  Lets get real and down to earth.  This makes me see Github in a way worse light, as support seems utterly inaccessible in light of a data corrupting bug in your Git client, running on Mac m2, latest version: 3.2.3

Actual Ticket:
Notice: no images like .png over 20mb upload.  This isnt a Large file.  But the service blocks them without erroring in the client.  This happens with the Latest version of Github Desktop Client.   Its not doing anything intelligent but more obfuscating.  If I rename the .png to .hello it works.  So it isnt some intelligent filter but more like a script running in your system that blocks any form of error returned to the client during the push process.  This is particularly bad because people get nervous during that process.

I have tested the same exact png on a brand new repo and it is clearly failing silently with the .png extension.  This is your network, and its a corrupted transaction with no status as to what is blocking it.  This happens during the push process.  I am ONLY using your desktop client here.  Not doing anything silly between two git utilities.

Resolution:
Report a user error message on this.  My tests include trying in a brand new repo on another machine, using the latest version.  Sure, you SHOULDNT upload large images OFTEN.  Don't bash me with a sock and say its not source code.  I am saying now that this isnt a stable product and not a clear transaction taking place.  No user errors on this?  This is a corrupting product experience, and one that flocks people away from Github.   That should Probably matter to you, as this is just about as common a task as people can do with repos and minor resources under the LFS limit of 100 MB.

My current consumer path:
Moving away from github soon.  There is clear intent to harm your users data here.  No status output during this push process, no sense of what is holding the process up.  Thats a faulty design especially when the only actionable thing you can do is force quit the app.

Inaccessibility, thwarting the user with intentional support traffic misdirection:
The way you get to this page is a tacky design.  Seems like a major Redmond movie being played backwards.  There is no support page for the Github Client Desktop.  I now rank this product 1 out of 5 stars, as its trying to delete my data.  Thats disturbing because its my work and something as simple as resource handling with any given file is now suspect due to your criteria you dont expose, or give status to for the user.

**Comment by siminapasat** [comment]
Hi @ar-visions 👋 thanks for reporting this issue and sorry you are facing it.

GitHub Desktop is an open source project and bugs are reported to http://github.com/desktop/desktop - if you click on the Help menu in the Desktop app, and then on "Report an issue", you'll be redirected to the open source project where you can file an issue.

**Comment by steveward** [comment]
@ar-visions I'm sorry for the frustration here. Like you mentioned GitHub blocks files that are over 100MB in size, but 20MB should not be an issue. I tested this with a 30MB PNG image file and was unable to reproduce this specific issue, so my guess is there is another underlying problem preventing the push from completing. If you'd like to open an issue at https://github.com/desktop/desktop/issues we can debug things further there, or you can share a log file in a reply here. The log file should give a better indication of what's happening in this specific case.

To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs...`. The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue. The push operation should look something like this in the log:

```
2023-05-10T15:25:40.964Z - info: [ui] Executing push: git -c credential.helper= push origin main:main --progress (took 9.287s)
```

It'd also be helpful to know if you are using HTTPS or SSH with this specific repository. 

>No status output during this push process, 

You should see status output like this unless the push is hanging:

![](https://user-images.githubusercontent.com/134360/54933084-35438d00-4ed9-11e9-9e56-b35165918671.png)

If the push is hanging then you should see a spinner icon showing the operation is stuck.
